olive/cache.py

```diff
@@ -148,6 +145,21 @@ def get_non_local_resource(resource_path: ResourcePath, cache_dir: Union[str, Pa
     return local_resource_path
 
 
+def get_local_path(resource_path: ResourcePath, cache_dir: Union[str, Path] = ".olive-cache"):
```

Comment:
>cache_dir [](http://example.com/codeflow?start=48&length=9)

this parameter seems useless? if so, we could make the function as method of ResourcePath?